packages/storybook8/stories/Components/CaptionsSettingsModal/CaptionsSettingsModal.story.tsx

```diff
@@ -53,31 +53,40 @@ const CaptionsSettingsModalStory = (args: any): JSX.Element => {
     'sk-sk',
     'zh-tw'
   ];
-
+  const supportedCaptionLanguages: Array<keyof CaptionLanguageStrings> = [
+    'ar',
+    'da',
+    'de',
+    'en',
+    'es',
+    'fi',
+    'fr',
+    'fr-ca',
+    'hi',
+    'it',
+    'ja',
+    'ko',
+    'nb',
+    'nl'
+  ];
   const currentSpokenLanguage = 'en-us';
+  const currentCaptionLanguage = 'en';
   const [showModal, setShowModal] = useState<boolean>(false);
   const onDismissCaptionsSettings = (): void => {
     setShowModal(false);
   };
   const onSetSpokenLanguage = (language: any): Promise<void> => {
-    alert(`Selected ${language}`);
+    console.log(`Spoken language set to ${language}`);
```

Comment:
Let's remove console.logs

Reply:
This is to make the storybook page a better experience. Alert can be jarring while console.log is quieter.

Reply:
Updated to use alert instead